### PR TITLE
closes #35 closes #31 advs vignette

### LINKED ISSUE #31: WHO metadata add a note on extending beyond 5 years
David shared this resource for WHO data beyond 5 years: https://www.who.int/tools/growth-reference-data-for-5to19-years/indicators

We could add a note to our site pages for those applicable metadata to share this link if ever needing to cover beyond 5 years old.

@Pierre-Wallet - you might want to take this on for continuity?

### LINKED ISSUE #35: ADVS growth parameter vignette
See MS teams outline doc and refer to ADVS template.

Reminder to explain where users could provide their own or varying metadata, such as using only WHO (instead of CDC <2yrs), or WHO extended beyond 5 yrs, or even IOTF (Intl Obesity Task Force) standards: http://www.ncbi.nlm.nih.gov/pubmed/22715120

Also update the Get Started page in same issue/PR.

### PR DESCRIPTION
**Pull request by rossfarrugia**
**NOTE the main part of this PR is to add the new ADVS vignette, but I also added the Get Started guide, updated the logo favicons and made a few other minor tweaks from what we started with from the default admiraltemplate.**

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiralpeds (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiralpeds site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@zdz2101 i noticed that nowhere in the function documentation or the vignette do we explain the formulae used for Z-score and percentile derivations. do you think this should be explained in the function roxygen2 header sections? or we're only going to get this question from users.

**Comment by rossfarrugia** [comment]
> I'll add it to the documentation using this branch, perhaps we should include it in the vignette as well

yes i was wondering that too, but then i added a reference to the 2 functions used, so my hope was any user would click those and then read direct from the function pages. but yeh when you come back to review this vignette then see if you think worth to also add something here.

**Review by zdz2101 — APPROVED** [review]
LGTM

**Review by Lina2689 — APPROVED** [review]
Looks good!

**Comment by rossfarrugia** [comment]
@Fanny-Gautier once you're happy with this one i'm happy to merge! 
Also anyone else in the team feel free to review.

File: vignettes/advs.Rmd [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+library(admiraldev)
+```
+
+# Introduction
+
+This article describes creating a vital signs ADaM for pediatric
+clinical trials. The main focus is around Anthropometric indicators
+(i.e. growth development chart) parameter derivations.
+
+We advise you first consult the `{admiral}` [Creating a BDS Finding ADaM
```
Comment (Fanny-Gautier) [inline vignettes/advs.Rmd:26]: Not sure why but it appears as 2 different links.
image

File: vignettes/advs.Rmd [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+  bind_rows(cdc_bmiage %>%
+    mutate(
+      SEX = case_when(
+        SEX == 1 ~ "M",
+        SEX == 2 ~ "F",
+        TRUE ~ NA_character_
+      ),
+      AGE = AGE * 30.4375
+    )) %>%
+  # AGEU is added in metadata, required for derive_params_growth_age
```
Comment (Fanny-Gautier) [inline vignettes/advs.Rmd:115]: Can we add the brackets in the comment for the corresponding function `derive_params_growth_age()` ?

File: vignettes/advs.Rmd [hunk 3]
```diff
@@ -0,0 +1,491 @@
...
+  mutate(AGEU = "DAYS") %>%
+  arrange(AGE, SEX)
+```
+
+Here is how the combined metadata now looks for the age ranges around 2
```
Comment (Fanny-Gautier) [inline vignettes/advs.Rmd:120]: combined metadata for BMI ? at line 136 we are suddenly talking about HDC

File: vignettes/advs.Rmd [hunk 4]
```diff
@@ -0,0 +1,491 @@
...
+)
+```
+
+To help interpret this metadata: the LMS parameters are the median (M), the
+generalized coefficient of variation (S), and the power in the Box-Cox
+transformation (L). Sigma is the dispersion parameter used in the calculation of
+BMI percentiles and z-scores above the 95th percentile, and P95 represents this
+sex and age-specific 95th percentile.
+
+Note that the head circumference metadata comes only from the WHO standards,
```
Comment (Fanny-Gautier) [inline vignettes/advs.Rmd:136]: it's confusing me as we were talking about BMI metadata as an example in this section

File: vignettes/advs.Rmd [hunk 5]
```diff
@@ -0,0 +1,491 @@
...
+functions we took the simple approach to use the nearest age from the
+metadata, rather than apply some form of interpolation. An alternative
+method you could use would be to derive current age according to the
+exact same intervals as the metadata in order for an exact join, but in
+our examples we just kept this by days.
+
+### Derived Variables for Parameters by Age
+
+A variable for current analysis age could be achieved using the
+following code (you might first need to derive `BRTHDT` if not already
```
Comment (Fanny-Gautier) [inline vignettes/advs.Rmd:293]: Should we mention that BRTHDT must be not missing ? There are countries were the full birth date can't be collected in the CRF. it's to the programming team to impute it...

File: vignettes/advs.Rmd [hunk 6]
```diff
@@ -0,0 +1,491 @@
...
+    new_vars = exprs(HGTTMP = AVAL, HGTTMPU = VSSTRESU)
+  )
+```
+
+## Derive Additional Parameters for Anthropometric indicators {#derive_params}
+
+Now we get to the most important section where we show how to create the
+new records for each derived pediatrics parameter. Only certain examples
+are included here, but you'll find more parameters in the example
+template script.
```
Comment (Fanny-Gautier) [inline vignettes/advs.Rmd:337]: Can you provide the link ?

File: vignettes/advs.Rmd [hunk 7]
```diff
@@ -0,0 +1,491 @@
...
+
+Here it is important once again to remind of the importance of the
+metadata being supplied to the above function via the `meta_criteria`
+argument. In our default case from this package this is taking WHO
+standards for children \<2 years of age and CDC standards for children
+\>=2 years of age. You as the user are free to supply any other choice
+of metadata here from the reference files you prefer.
+
+For BMI parameters only, you'll notice from the code above that we have
+set `bmi_cdc_correction = TRUE`. This is because we opted to use the CDC
```
Comment (Fanny-Gautier) [inline vignettes/advs.Rmd:379]: What is the other choice and its impact ?

**Review by Fanny-Gautier — COMMENTED** [review]
Review of the vignette completed: Minor comments regarding the understanding.

File: vignettes/advs.Rmd (hunk 1 above)
Comment (rossfarrugia) [reply vignettes/advs.Rmd:26]: Same occurs in admiralonco as below so will leave it. Think the first links to the admiral homepage in case people are completely new to the package and the other directly to the vignette:

image

File: vignettes/advs.Rmd (hunk 2 above)
Comment (rossfarrugia) [reply vignettes/advs.Rmd:117]: sure, will do

File: vignettes/advs.Rmd (hunk 3 above)
Comment (rossfarrugia) [reply vignettes/advs.Rmd:122]: will try to explain this better to make it clearer 👍

File: vignettes/advs.Rmd (hunk 4 above)
Comment (rossfarrugia) [reply vignettes/advs.Rmd:138]: will clarify

File: vignettes/advs.Rmd (hunk 5 above)
Comment (rossfarrugia) [reply vignettes/advs.Rmd:295]: sure, will explain

File: vignettes/advs.Rmd (hunk 6 above)
Comment (rossfarrugia) [reply vignettes/advs.Rmd:339]: ok, will add `referenced at the bottom of this vignette` rather than a link, as people should always use `use_ad_template()` to get this.

File: vignettes/advs.Rmd (hunk 1 above)
Comment (Fanny-Gautier) [reply vignettes/advs.Rmd:27]: ok, let's keep it as in admiralonco.

**Review by Fanny-Gautier — APPROVED** [review]
No more comments. Thanks for the updates
